wscripts/wscript

```diff
@@ -0,0 +1,41 @@
+
+#
+# This file is the default set of rules to compile a Pebble project.
+#
+# Feel free to customize this to your needs.
+#
+
+import os
+import os.path
+
+
+top = '.'
+out = 'build'
+
+def options(ctx):
+    ctx.load('pebble_sdk')
+
+def configure(ctx):
+    ctx.load('pebble_sdk')
+
+def build(ctx):
+    ctx.load('pebble_sdk')
+
+    build_worker = os.path.exists('worker_src')
```

Comment:
I believe so, but this is just a copy of the default wscript that is distributed with the SDK. Will update it anyway and add a comment